static/app/views/starfish/views/appStartup/screenSummary/index.tsx

```diff
@@ -96,101 +97,104 @@ function ScreenSummary() {
           <Layout.Body>
             <Layout.Main fullWidth>
               <PageAlert />
-              <PageFiltersContainer>
-                <Container>
-                  <PageFilterBar condensed>
-                    <DatePageFilter />
-                  </PageFilterBar>
+              <HeaderContainer>
+                <ControlsContainer>
```

Comment:
This diff is a result of shift around the closing tags that had more children than was intended